src/esp/physics/CollisionGroupHelper.h

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_PHYSICS_COLLISIONGROUPHELPER_H_
+#define ESP_PHYSICS_COLLISIONGROUPHELPER_H_
+
+#include <Corrade/Utility/Assert.h>
+#include <Magnum/Magnum.h>
+#include <map>
+#include <string>
+#include <vector>
+#include "esp/core/Check.h"
+#include "esp/core/esp.h"
+
+/** @file
+ * @brief Class @ref esp::physics::CollisionGroupHelper is a convenience class
+ * with all static functions providing an interface for customizing collision
+ * masking behavior for simulated objects. Enum Class @ref
+ * esp::physics::CollisionGroup defines available collision groups.
+ */
+
+namespace esp {
+namespace physics {
+
+/*
+  For reference, here are the standard Bullet groups. It's probably good to
+  stay somewhat consistent with them, in case users are already using them.
+        enum CollisionFilterGroups
+        {
+                DefaultFilter = 1,
+                StaticFilter = 2,
+                KinematicFilter = 4,
+                DebrisFilter = 8,
+                SensorTrigger = 16,
+                CharacterFilter = 32,
+                AllFilter = -1  //all bits sets: DefaultFilter | StaticFilter |
+  KinematicFilter | DebrisFilter | SensorTrigger
+        };
+*/
+
+/**
+ * @brief Defined available collision groups with bitwise integer correspondance
+ * for use in collision filtering masks.
+ *
+ * Contains several named default groups used by Habitat-sim and a set of unused
+ * user groups for custom behavior.
+ */
+enum class CollisionGroup {
```

Comment:
Optional: This makes it clear this could be a Corrade EnumSet https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1EnumSet.html with all the convience functions you get from that.

Reply:
At the very least, it can inherit from an unsigned int.